File: Exercise/duplicate_remove.c
#include<stdio.h>
#include<stdlib.h>
int main()
{

int a[50],i,j,k, count = 0, n;
   printf("Enter size of the array");
   scanf("%d",&n);
   printf("Enter Elements of the array:");
   for(i=0;i<n;i++){
      scanf("%d",&a[i]);
   }


   for(i=0;i<n;i++)
   {
    for(j=i+1;j<n;j++)
    {
        if(a[i]==a[j])
        {
            for(k=j;k<n;k++)
            {
                a[k]=a[k+1];
            }
            j--;
            n--;
        }
    }
   }

   for(i=0;i<n;i++){
      printf("%d",a[i]);
   }
}

File: Exercise/duplicate_remove.py
n=int(input("Enter the no. of elements:"))
a=list(map(int,input("Enter the numbers:").strip().split()))[:n]

for i in range(0,n-1):
    for j in range(i+1,n):
        if(a[i]==a[j]):
            for k in range(j,n-1):
                a[k]=a[k+1]
            n=n-1
            j=j-1

print(a)